test/math/FloatComplexVectorListTest.ooc

```diff
@@ -34,7 +34,7 @@ FloatComplexVectorListTest: class extends Fixture {
 	complexNumberArray add(complexNumber2)
 	complexNumberArray add(complexNumber3)
 
-	tolerance := 0.00001
+	tolerance := 0.00001f
```

Comment:
Looks like Constraints can't handle different parameter types for tolerance and float arguments.
This change fixed the uninitialized memory read.